src/client/src/orangehrmRecruitmentPlugin/pages/ShortlistCandidateScreen.vue

```diff
@@ -0,0 +1,182 @@
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+<template>
+  <div class="orangehrm-background-container orangehrm-save-candidate-page">
+    <div class="orangehrm-card-container">
+      <oxd-text tag="h6" class="orangehrm-main-title">
+        Shortlist Candidate
+      </oxd-text>
+      <oxd-divider />
+      <oxd-form :loading="isLoading" @submitValid="onSave">
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Candidate"
+                disabled
+                :value="shortlist.candidate"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Vacancy"
+                disabled
+                :value="shortlist.vacancy"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Hiring Manager"
+                disabled
+                :value="shortlist.manager"
+              />
+            </oxd-grid-item>
+            <oxd-grid-item>
+              <oxd-input-field
+                label="Current Status"
+                disabled
+                :value="shortlist.status"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item class="orangehrm-save-candidate-page-full-width">
+              <oxd-input-field
+                v-model="shortlist.note"
+                label="notes"
+                type="textarea"
+                :rules="rules.note"
+              />
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+        <oxd-divider />
+        <oxd-form-actions>
+          <required-text />
+          <oxd-button display-type="ghost" :label="$t('general.back')" />
+          <submit-button label="Shortlist" />
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {shouldNotLessThanCharLength} from '@ohrm/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+import {navigate} from '@/core/util/helper/navigation';
+export default {
+  name: 'ShortlistCandidateScreen',
+  setup() {
+    const http = new APIService(
+      'https://01eefc6d-daf1-4643-97ae-2d15ea8b587b.mock.pstmn.io',
+      'recruitment/shortlistCandidate',
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      rules: {
+        notes: [shouldNotLessThanCharLength(250)],
+      },
+      shortlist: {
+        candidate: '',
+        vacancy: '',
+        manager: '',
+        status: '',
+        note: '',
+      },
+      statuses: [
```

Comment:
is this not used in select?

Reply:
no but in the API we only get Id.

Reply:
@SandeepaOHRM can't we get the status object? hard-coding in FE doesn't feel right

Reply:
Me and Ajiharan discussed this. We do have all these action status in the workflow state machine. getting it from through API feels right for me rather hard-coding like this format.
`status:[
{
'label':'Application Initiated';
'status': 1
},
{
'label':'Shortlisted',
'status':2
},
...
]`

Reply:
`status:[
{
'label':'Application Initiated';
'status': 1
},
{
'label':'Shortlisted',
'status':2
},
...
]`

Reply:
@Ajiharan007 then please make a mock API and add here.

Reply:
@Super-Chama yeah for the moment lets go with mock API. I will create a seperate API for this